### ISSUE TITLE
Print the version number and commit hash for haskell-ci in generated .travis.yml files

### ISSUE THREAD

**Issue by RyanGlScott** [haskell-CI/haskell-ci#237]
When `haskell-ci`-generated `.travis.yml` scripts go wrong, it's sometimes difficult to tell if it's due to a bug in an old version of `haskell-ci` or not. The reason for this is that `haskell-ci`'s generated `.travis.yml` files give no indication whatsoever of what version of `haskell-ci` was used to generate it, which forces me to guess.

To make this easier, I think it would be nice if `haskell-ci` printed version information at the top of each `.travis.yml` file, similarly to what `hpack` does for its generated `.cabal` files:

```
-- This file has been generated from package.yaml by hpack version 0.31.0.
--
-- see: https://github.com/sol/hpack
--
-- hash: cbcda7d66dae228545cd78a42c4c4edf8ec2a746587f3aac59ba8a8775252cc5
```

**Comment by phadej** [comment]
Note, that with `v2-install` `gitrev` doesn't work, as there aren't `.git` in sdist `tar.gz`.

**Comment by RyanGlScott** [comment]
Very true. I'd at least be happy with the version number, as that's better than nothing.

**Comment by phadej** [comment]
Also note that `hash: ` in hpack is hash of the generated contents

https://github.com/sol/hpack/blob/5d847732774c1ea9bd5d5b52125c11fa01de4e60/src/Hpack.hs#L176-L185

**Comment by RyanGlScott** [comment]
Fixed in #239.
